seed/ts-sdk/imdb/no-custom-config/src/api/resources/imdb/client/Client.ts

```diff
@@ -43,10 +43,29 @@ export class Imdb {
      *         rating: 1.1
      *     })
      */
-    public async createMovie(
+    public createMovie(
         request: SeedApi.CreateMovieRequest,
         requestOptions?: Imdb.RequestOptions,
-    ): Promise<SeedApi.MovieId> {
+    ): core.HttpResponsePromise<SeedApi.MovieId> {
+        return core.HttpResponsePromise.fromFunction(this.__createMovie, request, requestOptions);
+    }
+
+    /**
+     * Add a movie to the database using the movies/* /... path.
+     *
+     * @param {SeedApi.CreateMovieRequest} request
+     * @param {Imdb.RequestOptions} requestOptions - Request-specific configuration.
+     *
+     * @example
+     *     await client.imdb.createMovie({
+     *         title: "title",
+     *         rating: 1.1
+     *     })
+     */
```

Comment:
I wonder if we should _not_ include the docs on the private method (since it's otherwise redundant. Thoughts?